modules/schema/src/main/resources/lucuma/odb/graphql/OdbSchema.graphql

```diff
@@ -9098,6 +9205,12 @@ type GmosNorthImaging {
   """
   offsets: [Offset!]!
 
+  "Temporary, WIP -- will be moved."
+  objectOffsetGenerator: OffsetGenerator
```

Comment:
Note the `offsets` field above would be removed and the generators themselves placed into new imaging observation sub-types 😢.  I didn't want to break anything yet though.